src/config/scrivitoContentBrowser.js

```diff
@@ -1,6 +1,8 @@
 import * as Scrivito from "scrivito";
 
 export function configureScrivitoContentBrowser() {
+  if (typeof window === "undefined") return;
```

Comment:
* [ ] FYI The `configureContentBrowser` API is available under Node, it just has no effect (obviously). For simplicity, we could remove the early-exit condition.